api/SharedStorage.json

```diff
@@ -75,6 +75,40 @@
           }
         }
       },
+      "batchUpdate": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/shared-storage/#dom-sharedstorage-batchupdate",
+          "support": {
+            "chrome": {
+              "version_added": "134"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror",
```

Comment:
This appears to be disabled in WebView, see:

https://github.com/chromium/chromium/blob/671ef42df9ed128258d1793a38c74c856c93d812/android_webview/browser/aw_field_trials.cc#L124-L125

Reply:
Great catch, thank you! I updated this for the new Shared Storage features that this PR is adding. We will need to update the rest in a separate PR. See also https://github.com/mdn/browser-compat-data/issues/24690 which filed this as an issue.